[overall] fixes some bugs (#1)
inclyc/LevelTSDB-doc: sections/introduction.tex

The Internet of Things(IoT) technology now penetrates various scenarios today.
Examples include smart healthcare, industrial monitoring, and city-scale

> sensing. A classic scenario is that a set of sensors keep tracking and focus on

Reviewer: ```suggestion
sensing. A classic scenario is that a set of sensors keep tracking and focusing on
```

> its metric, then upload to dedicated cloud server with timestamps. The cloud

Reviewer: ```suggestion
its metric, then upload to a dedicated cloud server with timestamps. The cloud
```

> server will analyze these multi-source data in order to generate knowledge like

Reviewer: ```suggestion
server will analyze these multi-source data to generate knowledge like
```

system status.

% 第二段 时序数据库的聚合、分析带来了什么挑战 -- 快速的查询聚合值
In this context, there exists a general need to store time-series data and
quickly retrieve it for cloud server analysis, and the database provides a
caching mechanism that enables quick queries for some common types of analysis.
In relational database practice, queries based on aggregation operations such as

> sum, maximum, and so on are very common, and are widely used in monitoring and

Reviewer: ```suggestion
sum, maximum, and so on are very common and are widely used in monitoring and
```

% 第七段 本文的结构

> In this article, we present a new data structure focus on optimization of

Reviewer: ```suggestion
In this article, we present a new data structure focus on the optimization of
```

> aggregate operations. And we analysis time complexity and perform benchmark to

Reviewer: ```suggestion
aggregate operations. And we analyzed time complexity and perform a benchmark to
```

> show the advanced nature of this data structure in comparison of other popular time

Reviewer: ```suggestion
show the advanced nature of this data structure in a comparison of other popular time
```

Reviewer: Grammerly说的(笑)

Reviewer: 会不会改成`in comparison with`好点

Author: 草，刚又看了一边，搭配是`in comparison with`

Author: `comparsion`可做可数名词，有`a comparison of xxx`，不过这里一般`comparison`做主语

Author: 喵喵
